doc/ip/pinmux.md

```diff
@@ -2,11 +2,21 @@
 
 This allows software to dynamically switch FPGA pins between input and output as well as reassign them for SPI, I2C, UART, etc.
 The block also allows pad control.
-An example use-case is changing whether an LED is driven by the GPIO or by the PWM.
 
-We will use [the pin multiplexer from OpenTitan](https://opentitan.org/book/hw/ip/pinmux/index.html), but then parameterized for Sonata.
-More details on how this changes the register map will be provided in the future.
+To see the possible mappings, please refer to [the top configuration](https://github.com/lowRISC/sonata-system/blob/main/data/top_config.toml).
 
-In general, we expect each pin can be connected to a limited subset of blocks.
-The output of GPIO and PWM should only be able to control one pin, while blocks like UART, SPI and I2C should have multiple pins that they can connect to.
-The exact mapping of which pin can be driven by which hardware IP block will be provided in the future.
+Register addresses `0x000` to `0x7ff` are pin selectors, which are packed 4 per 32-bit word.
+Until we have an automated way to generate documentation, you'll have to look in [the generated pinmux RTL](https://github.com/lowRISC/sonata-system/blob/main/rtl/system/pinmux.sv)
```

Comment:
Once this is in and settled, we should document the default configuration. Mention of other top configurations should probably be left out of the reference manual and be somewhere else, somewhat hidden from normal users.

Reply:
The table for this is automatically generated now and it mentions the default selected value is `'b10`